### DISCUSSION TITLE
Problem with SFLOW on a Cisco Nexus 9500

### DISCUSSION THREAD

**Discussion by gautier0102** [akvorado/akvorado#958] [Q&A]
Hi,

I'm sending flow trough SFLOW from a Cisco Nexus 9500, I have 3 VRFs  (default one, and two others). I don't need VRF information on Akvorado but flow coming from a VRF aren't correctly imported on AKVORADO.

I have this on the flow API :

`{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369102459,"OutIf":369098868,"SrcVlan":0,"DstVlan":0,"SrcAddr":"::ffff:58.27.137.250","DstAddr":"::ffff:213.5.130.16","NextHop":"","SrcAS":38264,"DstAS":34534,"GotASPath":false,"SrcNetMask":24,"DstNetMask":24}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098855,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098859,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098866,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098859,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098870,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098860,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}
{"TimeReceived":1700334451,"SamplingRate":4096,"ExporterAddress":"::ffff:10.104.198.2","InIf":369098859,"OutIf":369102459,"SrcVlan":0,"DstVlan":0,"SrcAddr":"","DstAddr":"","NextHop":"","SrcAS":0,"DstAS":0,"GotASPath":false,"SrcNetMask":0,"DstNetMask":0}`

And after some digging on a PCAP, I found the only difference on the flow are a VN-Tag on the header of the sampled packet when the packet is coming from non default VRF. Could we edit something to ignore this tag and import those flow on Akvorado ? 

<img width="888" alt="Capture d’écran 2023-11-18 à 19 51 19" src="https://github.com/akvorado/akvorado/assets/18391856/94dfb19e-f2e3-442a-a90f-5edd82282922">
<img width="865" alt="Capture d’écran 2023-11-18 à 19 50 35" src="https://github.com/akvorado/akvorado/assets/18391856/b7f52d77-785d-4e2c-b3d6-d44157d3ac47">

**Comment by vincentbernat** [comment]
I would need this PCAP to understand what is in the VN-Tag. @lspgn did you run into this yet?

**Reply by lspgn** [reply]
Haven't seen this use-case before. Is its own encapsulation? (rfc7364)
